src/librustc_codegen_ironox/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+authors = ["Gabriela Alexandra Moldovan"]
```

Comment:
I think you're supposed to add an email address? I might be wrong.

Reply:
Fixed!